---
layout: page
title: Publications
description: Publications
---

### Recent papers 
For a complete list, please see [Google Scholar](https://scholar.google.ca/citations?hl=en&user=Ohu9Gc4AAAAJ&view_op=list_works&sortby=pubdate). 
  
   
**Preprints** 

Lypaczewski P et al. (2024). Diversity of *Vibrio cholerae* O1 through the human gastrointestinal tract during cholera. [bioRxiv](https://www.biorxiv.org/content/10.1101/2024.02.08.579476v1.abstract).

Sutcliffe SG et al. (2023). Tracking SARS-CoV-2 variants of concern in wastewater: an assessment of nine computational tools using simulated genomic data. [bioRxiv](https://www.biorxiv.org/content/10.1101/2023.12.20.572426v1.abstract).

Madi N et al. (2023). Phage predation is a biomarker for disease severity and shapes pathogen genetic diversity in cholera patients. [bioRxiv](https://www.biorxiv.org/content/10.1101/2023.06.14.544933v2).

N'Guessan A et al. (2022) Detection of prevalent SARS-CoV-2 variant lineages in wastewater and clinical sequences from cities in Québec, Canada. [medRxiv](https://www.medrxiv.org/content/10.1101/2022.02.01.22270170v1).

Chen PE & Shapiro BJ (2021). Classic genome-wide association methods are unlikely to identify causal variants in strongly clonal microbial populations. [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.06.30.450606v1). 

**2024**  
Douglas GM & Shapiro BJ. Pseudogenes act as a neutral reference for detecting selection in prokaryotic pangenomes. [Nature Ecology & Evolution](https://www.nature.com/articles/s41559-023-02268-6).
* [*News & Views*](https://www.nature.com/articles/s41559-023-02276-6)

**2023**  
N'Guessan A et al. Selection for immune evasion in SARS-CoV-2 revealed by high-resolution epitope mapping and sequence analysis. [iScience](https://www.sciencedirect.com/science/article/pii/S2589004223014712). 

Madi N, Chen D, Wolff R, Shapiro BJ & Garud N. Community diversity is associated with intra-species genetic diversity and gene loss in the human gut microbiome. [eLife](https://elifesciences.org/articles/78530). 

Naderi S, Chen PE, Murall CL, Poujol R, Kraemer SA, Pickering BS, Sagan SM & Shapiro BJ. Zooanthroponotic transmission of SARS-CoV-2 and host-specific viral mutations revealed by genome-wide phylogenetic analysis. [eLife](https://elifesciences.org/articles/83685)

Saber MM, Donner M et al. Single nucleotide variants in *Pseudomonas aeruginosa* populations from sputum correlate with baseline lung function and predict disease progression in individuals with cystic fibrosis. [Microbial Genomics](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000981) 

Douglas GM, Kim S, Langille MGI & Shapiro BJ. Efficient computation of contributional diversity metrics from microbiome data with *FuncDiv*. [Bioinformatics](https://doi.org/10.1093/bioinformatics/btac809).  

**2022**  
Leducq J-B et al. Fine-scale adaptations to environmental variation and growth strategies drive phyllosphere Methylobacterium diversity. [mBio](https://journals.asm.org/doi/full/10.1128/mbio.03175-21)  

Barbosa da Costa N et al. A glyphosate-based herbicide cross-selects for antimicrobial resistance genes in bacterioplankton communities. [mSystems](https://journals.asm.org/doi/full/10.1128/msystems.01482-21) 

Creasy-Marrazzo A, Saber MM et al. Genome-wide association studies reveal distinct genetic correlates and increased heritability of antimicrobial resistance in *Vibrio cholerae* under anaerobic conditions. [Microbial Genomics](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000905)  

**2021**   
Murall CL et al. A small number of early introductions seeded widespread transmission of SARS-CoV-2 in Quebec, Canada. [Genome Medicine](https://link.springer.com/article/10.1186/s13073-021-00986-9)  
 
Douglas GM & Shapiro BJ. Genic selection within prokaryotic pangenomes. [Genome Biology & Evolution](https://doi.org/10.1093/gbe/evab234)  

Levade I, Saber MM, Midani F, Chowdhury F, Khan AI, Begum YA, Ryan ET, David LA, Calderwood SB, Harris JB, LaRocque RC, Qadri F, Shapiro BJ, Weil AA. Predicting *Vibrio cholerae* infection and disease severity using metagenomics in a prospective cohort study. [The Journal of Infectious Diseases](https://academic.oup.com/jid/article/223/2/342/5866141) 

Levade I et al. A combination of metagenomic and cultivation approaches reveal hypermutator phenotypes within Vibrio cholerae infected patients. [mSystems](https://journals.asm.org/doi/full/10.1128/mSystems.00889-21)   

Barbosa da Costa N, V Fugère, M Hébert, CC Xu, RD Barrett, B Beisner, G Bell, V Yargeau, GF Fussmann, A Gonzalez, BJ Shapiro. Resistance, resilience, and functional redundancy of freshwater microbial communities facing multiple agricultural stressors in a mesocosm experiment. [Molecular Ecology](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.16100)  

Olga M. Pérez-Carrascal, N Tromas, Y Terrat, E Moreno, A Giani, Laisa CB Marques, N Fortin, BJ Shapiro. Single-colony sequencing reveals microbe-by-microbiome phylosymbiosis between the cyanobacterium Microcystis and its associated bacteria. [Microbiome](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01140-8)  

N'Guessan A, Brito IL, Serohijos AWR, Shapiro BJ. Mobile gene sequence evolution within individual human gut microbiomes is better explained by gene-specific than host-specific selective pressures. [Genome Biology & Evolution](https://doi.org/10.1093/gbe/evab142)  

**2020**  
Madi N, Vos M, Murall CL, Legendre P, Shapiro BJ. Does diversity beget diversity in microbiomes? [eLife](https://elifesciences.org/articles/58999)  

Saber MM, Shapiro BJ. Benchmarking genome-wide association study (GWAS) methods using simulated genomes and phenotypes. [Microbial Genomics](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000337)  

Tromas N, Castelli M, Taranu ZE, Pimentel J, Pereira D, Marcoz R, Giani A, Shapiro BJ. The evolution of realized niches in freshwater *Synechococcus*. [Environmental Microbiology](https://ami-journals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.14930)  

Terrat Y, Farnaes L, Bradley J, Tromas N, Shapiro BJ. Two cases of type-a Haemophilus influenzae meningitis within the same week in the same hospital are phylogenetically unrelated but recently exchanged capsule genes. [Microbial Genomics](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000348)  

